sdk/storage/storage-blob/src/BlobQueryResponse.browser.ts

```diff
@@ -0,0 +1,499 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { HttpResponse } from "@azure/core-http";
+
+import {
+  BlobDownloadResponseModel,
+  BlobType,
+  CopyStatusType,
+  LeaseDurationType,
+  LeaseStateType,
+  LeaseStatusType,
+  BlobDownloadHeaders,
+  BlobQueryResponseModel
+} from "./generatedModels";
+import { Metadata } from "./models";
+import { BlobQuickQueryStreamOptions } from "./utils/BlobQuickQueryStream";
+
+/**
+ * ONLY AVAILABLE IN BROWSER RUNTIME.
+ *
+ * BlobQueryResponse implements BlobDownloadResponseModel interface, and in browser runtime it will
+ * parser avor data returned by blob query.
```

Comment:
minor: should be avro, same to other places

Reply:
Good catch, will update.